app/src/renderer/vuex/modules/send.js

```diff
@@ -71,14 +81,25 @@ export default ({ node }) => {
 
       await dispatch(`queryWalletBalances`) // the nonce was getting out of sync, this is to force a sync
 
+      const gasPrices = [
+        {
+          amount: `0.025`, // recommended on Cosmos Docs
+          denom: `stake` //TODO: this shouldn't be hardcoded
```

Comment:
isn't this going to be an issue immediately?

Reply:
ohh true, good catch !